<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>475</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sawasdee</family>
    <pointsize>15</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(10, 10, 10);
font: 15pt &quot;Sawasdee&quot;;</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_sub">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 31, 49);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sum">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(36, 31, 49);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>121</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 22);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dot">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_percent">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 2, 33);</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_multi">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 31, 49);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_div">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 31, 49);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_C">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(150, 100, 0);</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sqrt">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 2, 33);</string>
    </property>
    <property name="text">
     <string>SQRT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_log">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 2, 33);</string>
    </property>
    <property name="text">
     <string>LOG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cot">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 2, 33);</string>
    </property>
    <property name="text">
     <string>cot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cos">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 2, 33);</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tan">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 2, 33);</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sin">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 2, 33);</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>302</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
